components/uuf-core/src/test/java/org/wso2/carbon/uuf/core/AppTest.java

```diff
@@ -68,6 +70,20 @@ private static Fragment createFragment(String name, String content) {
         return new Fragment(name, renderable, false);
     }
 
+    private static Fragment createFragmentWithResources(String name, String content) {
+        Renderable renderable = (model, l, rl, a) -> content + (model.toMap().isEmpty() ? "" : model.toMap());
```

Comment:
No need to create a Renderable here as we override the render method of the Fragment.